### PR TITLE
TAD MoneyHelper banner

### PR DESCRIPTION
**Pull request by akenzua**
[TP12492](https://maps.tpondemand.com/restui/board.aspx?#page=board/5175634625349962976&appConfig=eyJhY2lkIjoiOUM2MjFGQzAzOEIzQTVDNDU1NjRCMUIxQ0FENkZBOTYifQ==&boardPopup=userstory/12492/silent)

This PR creates the MoneyHelper banner in TAD

### CODE REVIEW

File: Utils/layouts/banner.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  `;
+
+
+const MoneyHelperBanner = ({ t }) => {
+   
+    return (
+        <BannerSection align="stretch" style={{ background: "#0F19A0" }} >
+            <SubSection constrained >
+                <Col>
+                    <Heading level={3} textSize="19px" lineHeight="0.5rem" color="#fff">{t("bannerHeading")}</Heading>
```
Comment (davidtrussler) [inline Utils/layouts/banner.js:43]: This heading needs a greater value for line-height as it becomes vertically over-condensed on (very) small viewports. The MAS equivalent is 1.5rem. Also the font-size on MAS is 1.125rem which resolves to 18px though this is maybe less critical.

File: Utils/layouts/banner.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  flex-direction: row;
+  ${resolveMedia.md`
+    justify-content: flex-end;
+ 
+`};
+  `;
+
+  export const LargeLogo = styled(MoneyHelperLogo)`
+  display: none;
+  ${resolveMedia.md`
```
Comment (davidtrussler) [inline Utils/layouts/banner.js:21]: I think we would need to render the large/inline logos at a smaller breakpoint to be consistent with the design, such as using `resolveMedia.sm`.

File: Utils/layouts/banner.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import { BannerSection, SubSection, BannerText } from "."
+import { i18n, withTranslation } from "../translation/i18n";
+import {Col, Heading, resolveMedia} from "@moneypensionservice/directories";
+import styled from "styled-components";
+
+import LogoEN from "./LogoEN.jsx"
+import LogoCY from "./LogoCY.jsx"
+
+export const MoneyHelperLogo = styled(Col)`
+  display: flex;
```
Comment (davidtrussler) [inline Utils/layouts/banner.js:11]: I don't think applying `display: flex` here makes any difference, if anything that should be applied to the parent element, though in fact the layout requires this element to float such that it doesn't appear below the text at some widths.

File: public/static/locales/cy/header.json [hunk 4]
```diff
@@ -9,5 +9,7 @@
   "listings": "Rhestrau",
   "home": "Hafan",
   "travel": "Cyfeiriadur Teithio",
-  "mobile": "Yn ôl i'r hafan"
+  "mobile": "Yn ôl i'r hafan",
+  "bannerHeading": "Rydyn yn symud i HelpwrArian",
+  "banner": "Bydd Pension Wise yn rhan o HelpwrArian, y ffordd hawdd o gael cymorth ar gyfer eich arian a phensiynau."
```
Comment (davidtrussler) [inline public/static/locales/cy/header.json:14]: Can we check the copy here? This appears to be the copy for Pension Wise where (I guess) we need the copy for MAS.

File: public/static/locales/en/header.json [hunk 5]
```diff
@@ -10,5 +10,8 @@
   "listings": "Listings",
   "home": "Home",
   "travel": "Travel Directory",
-  "mobile": "Back to homepage"
+  "mobile": "Back to homepage",
+  "bannerHeading": "We're moving to MoneyHelper",
+  "banner" :"The PensionWise service will soon be part of MoneyHelper, the easy way to get free help for your money and penions.",
```
Comment (davidtrussler) [inline public/static/locales/en/header.json:15]: Can we check the copy here? This appears to be the copy for Pension Wise where (I guess) we need the copy for MAS. (Also 'pensions' is misspelled)

**Comment by davidtrussler** [comment]
I've left a few comments regarding layout issues. 

In addition, I think this work should probably be done in the Library which may simplify the changes needed and also ensure we do not have unused components in the library and duplicated components in this project to enable potential future development. Given the short-term nature of the components in question I understand the approach taken though but it may be worth considering if that would be a simpler way to achieve what is needed here and what impact future requirements may have on the approach.

File: public/static/locales/en/header.json (hunk 5 above)
Comment (akenzua) [reply public/static/locales/en/header.json:15]: Fixed.

File: public/static/locales/cy/header.json (hunk 4 above)
Comment (akenzua) [reply public/static/locales/cy/header.json:14]: Fixed

File: Utils/layouts/banner.js (hunk 1 above)
Comment (akenzua) [reply Utils/layouts/banner.js:43]: Used 1 to constrain what was inherited

File: Utils/layouts/banner.js (hunk 3 above)
Comment (akenzua) [reply Utils/layouts/banner.js:11]: Ooops! Forgot to remove that

**Review by akenzua — COMMENTED** [review]
> I've left a few comments regarding layout issues.
> 
> In addition, I think this work should probably be done in the Library which may simplify the changes needed and also ensure we do not have unused components in the library and duplicated components in this project to enable potential future development. Given the short-term nature of the components in question I understand the approach taken though but it may be worth considering if that would be a simpler way to achieve what is needed here and what impact future requirements may have on the approach.

I thought so too but after discussing with @Andre, we agreed that doing it in the app will be the fastest way yo achieve that for now.

File: Utils/layouts/banner.js (hunk 2 above)
Comment (akenzua) [reply Utils/layouts/banner.js:21]: I don't think I understand what you meant here. We would probably pick this up tomorrow.

**Comment by davidtrussler** [comment]
> > I've left a few comments regarding layout issues.
> > In addition, I think this work should probably be done in the Library which may simplify the changes needed and also ensure we do not have unused components in the library and duplicated components in this project to enable potential future development. Given the short-term nature of the components in question I understand the approach taken though but it may be worth considering if that would be a simpler way to achieve what is needed here and what impact future requirements may have on the approach.
> 
> I thought so too but after discussing with @andre, we agreed that doing it in the app will be the fastest way yo achieve that for now.

Yes, the problem is that the existing banner is in the library so this approach duplicates the whole header. Maybe the simplest approach would be to delete the current banner from the library, then add the new one on this repo.

File: Utils/layouts/banner.js (hunk 2 above)
Comment (akenzua) [reply Utils/layouts/banner.js]: Fixed.

**Review by davidtrussler — APPROVED** [review]
LGTM! 👍

**Pull request merged**